### PR TITLE
BFD-747- Added carrier claim unit tests to v2

### PR DESCRIPTION
**Pull request by cbrunefearless**
### Change Details
User Story:

As a BFD engineer, I want there to be unit tests for the Carrier EOB profile, so that I can be confident that individual fields and values are working as intended.
Acceptance Criteria:

Confirm that the same level of and unit tests for BFD V2 are applied to V2
Ensure that all required CARIN data fields have corresponding unit tests for V2

### Acceptance Validation
Correct coverage of unit tests for EOB Carrier claims

### Feedback Requested
Verify unit tests pass and coverage is adequate.

### External References

- [BFD-747](https://jira.cms.gov/browse/BFD-747)

### Security Implications

- [ ] new software dependencies
N/A

N/A
- [ ] altered security controls

N/A
- [ ] new data stored or transmitted

N/A
- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### CODE REVIEW

File: apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CarrierClaimTransformerV2Test.java [hunk 1]
```diff
@@ -71,6 +100,944 @@ public void serializeSampleARecord() throws FHIRException {
...
+    Assert.assertTrue(compare.equalsDeep(quantity));
+  }
+
+  @Test
+  public void shouldHaveLineItemAdjudications() {
+    Assert.assertEquals(9, eob.getItemFirstRep().getAdjudication().size());
+  }
+
+  @Test
+  public void shouldHaveLineItemDenailReasonAdjudication() {
```
Comment (mackec1) [inline apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CarrierClaimTransformerV2Test.java:644]: minor spelling nit....

**Review by mackec1 — APPROVED** [review]
there was one minor spelling thing...I wouldn;t worry about it tho; LGTM.

**Review by rtravitz — COMMENTED** [review]
Looks good in general. Would just fix the name of the test that @mackec1 pointed out.

File: apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CarrierClaimTransformerV2Test.java (hunk 1 above)
Comment (cbrunefearless) [reply apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CarrierClaimTransformerV2Test.java]: fixed

**Review by rtravitz — APPROVED** [review]
LGTM ✨

**Review by mackec1 — APPROVED** [review]
LGTM

**Pull request merged**